### ISSUE TITLE
Need to Set default Value in ngx-mat-select-search 

### ISSUE THREAD

**Issue by sehgalneha** [bithost-gmbh/ngx-mat-select-search#147]
I have Used ngx-mat-select-search for append the user list according to our project requirement. Need to Set text on the first load of page. The List is shown as according to search text but unable to set text on the ngx-mat-select-search . Please check out snippet.
![image](https://user-images.githubusercontent.com/24697336/57525667-ecd91500-7348-11e9-97c2-28ae8f2e715b.png)

Can you please provide the solution to set text on ngx-mat-select-search.   I am looking forward for your response.

Thanks and Regards
Bhumika

**Comment by macjohnny** [comment]
can you please provide your sample code?
If I correctly understand your request, you are looking for a way to set the initial value of the search input field of `<ngx-mat-select-search>`?

Currently this is not possible, however it should be possible by adding a `[value]="value"` binding in
https://github.com/bithost-gmbh/ngx-mat-select-search/blob/d577317d5776ca109aff6738df77e71972c8e3fc/src/app/mat-select-search/mat-select-search.component.html#L9-L10

e.g.
```html
<input matInput
         class="mat-select-search-input"
         [value]="value"
```

@sehgalneha would you like to provide a PR that implements this?

**Comment by macjohnny** [comment]
@sehgalneha I filed https://github.com/bithost-gmbh/ngx-mat-select-search/pull/148 to allow setting the initial value as simple as
```typescript
public bankFilterCtrl: FormControl = new FormControl('my initial value');
```
Version 1.7.4 will be release soon

**Comment by sehgalneha** [comment]
public bankFilterCtrl: FormControl = new FormControl('my initial value');

According to this we have set value in our control :  "this.bankFilterCtrl.value" 
But it will be not displayed in the HTML.
Can you Please confirm this?

**Comment by macjohnny** [comment]
Yes, this should work after PR #148 being merged

**Comment by macjohnny** [comment]
this is released with version 1.7.4
```typescript
public bankFilterCtrl: FormControl = new FormControl('my initial search value');
```
See this example: https://stackblitz.com/edit/github-zwffsh?file=src/app/examples/01-single-selection-example/single-selection-example.component.ts
